src/System.IdentityModel.Tokens.Jwt/ClaimTypeMapping.tt

```diff
@@ -0,0 +1,115 @@
+﻿<#@ template debug="false" hostspecific="false" language="C#" #>
```

Comment:
This is a T4 file that generates the ClaimTypeMapping.cs from the ClaimsKnowledge.tti.